operators/olm_v1/index.adoc

```diff
@@ -0,0 +1,32 @@
+:_content-type: ASSEMBLY
+[id="olmv1-about"]
+= About Operator Lifecycle Manager v1 (Technology Preview)
+include::_attributes/common-attributes.adoc[]
+:context: olmv1-about
+
+toc::[]
+
+{product-title} 4.14 introduces components for a next-generation iteration of Operator Lifecycle Manager (OLM) as a Technology Preview feature. Known during this phase as OLM v1, the updated framework evolves many of the concepts that have been part of the version of OLM included since the release of {product-title} 4.
+
+:FeatureName: OLM v1
+include::snippets/technology-preview.adoc[]
+
+During this Technology Preview phase of OLM v1 in {product-title} 4.14, administrators can use file-based catalogs to install and manage the following:
+
+* OLM-based Operators, similar to the existing OLM experience
+* Plain bundles, which are static collections of arbitrary Kubernetes manifests
+
+[id="olmv1-about-purpose"]
+== Purpose
+
+The mission of Operator Lifecycle Manager (OLM) has been to manage the lifecycle of cluster extensions centrally and declaratively on Kubernetes clusters. Its purpose has always been to make installing, running, and updating functional extensions to the cluster easy, safe, and reproducible for cluster administrators and platform-as-a-service (PaaS) administrators, throughout the lifecycle of the underlying cluster.
+
+The existing version of OLM, which launched with {product-title} 4 and is included by default, was focused on providing unique support for these specific needs for a particular type of cluster extension, which have been referred as Operators. Operators are classified as one or more Kubernetes controllers, shipping with one or more API extensions (`CustomResourceDefinition` objects) to provide additional functionality to the cluster.
+
+After running OLM in production clusters for many releases, it became apparent that there is a desire to deviate from this coupling of CRDs and controllers to encompass the lifecycling of extensions that are not just Operators.
+
+Some of the goals of OLM v1 over the upcoming releases include improving Operator and extension lifecycle management in the following areas:
```

Comment:
[re: avoiding future claims] Checking with PM on how/if we want to make any statements like this. Included it here to try to give a sense that this release is just a part of what ultimately will feed into OLM v1's bigger picture when it's more fully-realized.

Reply:
Assuming you get approval on the specifics of the plan/claim, the language seems fine to me.